tests/functional/aws-node-sdk/test/object/websiteGetWithACL.js

```diff
@@ -0,0 +1,179 @@
+import fs from 'fs';
+import path from 'path';
+
+import async from 'async';
+import { S3 } from 'aws-sdk';
+import Browser from 'zombie';
+
+import conf from '../../../../../lib/Config';
+import getConfig from '../support/config';
+import { WebsiteConfigTester } from '../../lib/utility/website-util';
+
+const config = getConfig('default', { signatureVersion: 'v4' });
+const s3 = new S3(config);
+
+const transport = conf.https ? 'https' : 'http';
+const bucket = process.env.AWS_ON_AIR ? 'awsbucketwebsitetester' :
+    'bucketwebsitetester';
+const hostname = `${bucket}.s3-website-us-east-1.amazonaws.com`;
+
+const endpoint = process.env.AWS_ON_AIR ? `${transport}://${hostname}` :
+    `${transport}://${hostname}:8000`;
+
+const aclEquivalent = {
+    public: ['public-read-write', 'public-read'],
+    private: ['private', 'authenticated-read'],
+};
+
+const aclTests = [
+    // CEPH: test_website_private_bucket_list_private_index_blockederrordoc
+    {
+        it: 'should return 403 if private bucket index and error documents',
+        bucketACL: 'private',
+        objects: { index: 'private', error: 'private' },
+        html: '403-access-denied',
+    },
+    // CEPH: test_website_public_bucket_list_private_index_blockederrordoc
+    {
+        it: 'should return 403 if public bucket - private index - public ' +
+        'error documents',
+        bucketACL: 'public',
+        objects: { index: 'private', error: 'private' },
+        html: '403-access-denied',
+    },
+    {
+        it: 'should return index doc if private bucket - private index - ' +
```

Comment:
Yes - seems that a **public** object can be reached even though his bucket is private.
Tested against AWS.